Mathlib/Logic/Equiv/Pairwise.lean

```diff
@@ -0,0 +1,21 @@
+/-
+Copyright (c) 2024 Joseph Myers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joseph Myers
+-/
+import Mathlib.Data.FunLike.Equiv
+import Mathlib.Logic.Pairwise
+
+/-!
+# Interaction of equivalences with `Pairwise`
+-/
+
+lemma EquivLike.pairwise_comp_iff {X : Type*} {Y : Type*} {F} [EquivLike F Y X]
+    (f : F) (p : X → X → Prop) : Pairwise (fun y z ↦ p (f y) (f z)) ↔ Pairwise p := by
+  refine ⟨fun h ↦ fun i j hij ↦ ?_,
```

Comment:
I think there might be a shorter proof via `Surjective.forall`

Reply:
Now using `Function.Surjective.forall₂` (in the proof of one of the new unbundled lemmas).